tests/installation/installation_overview.pm

```diff
@@ -16,12 +16,24 @@ use warnings;
 use base "y2logsstep";
 use testapi;
 
+sub sle15_workaround_broken_patterns {
+    if (check_var('VERSION', '15')) {    # SLE 15 has pattern errors, workaround them - rbrown 04/07/2017
+        while (check_screen('sle-15-failed-to-select-pattern')) {
+            record_soft_failure 'bsc#1047327';
+            send_key 'alt-o';
+        }
```

Comment:
Your suggested approach will not work here - the select pattern error appears an almost random amount of times. Needs a while loop, sorry :(